src/components/Leaderboards.vue

```diff
@@ -53,6 +53,11 @@ const boardsSearched = computed(() =>
 	<div class="container">
 		<h1>Leaderboards</h1>
 		<div class="input-container">
+			<RouterLink :to="{ name: 'leaderboardsCreate' }">
+				<button class="button create-new">
+					Create New
+				</button>
+			</RouterLink>
```

Comment:
The proper way to do this is to do `<RouterLink tag="button">` I believe.

Reply:
`tag="button"` doesn't give the button UI, unfortunately.

Reply:
You're right; however, I did some more research and the actual proper way to do this is to simply style the link like a button. Gonna leave it for now though.

Reply:
Sure. I'm definitely not going to fuss over styling the anchor to look exactly like our buttons, matching colours and transitions and everything either, unless we really want to do that.